spring-boot-admin-server/src/test/java/de/codecentric/boot/admin/notify/SlackNotifierTest.java

```diff
@@ -0,0 +1,147 @@
+package de.codecentric.boot.admin.notify;
+
+import de.codecentric.boot.admin.event.ClientApplicationStatusChangedEvent;
+import de.codecentric.boot.admin.model.Application;
+import de.codecentric.boot.admin.model.StatusInfo;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.expression.Expression;
+import org.springframework.expression.ParserContext;
+import org.springframework.expression.spel.standard.SpelExpressionParser;
+import org.springframework.web.client.RestTemplate;
+
+import javax.annotation.Nullable;
+import java.net.URI;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class SlackNotifierTest {
+	private final String channel = "channel";
+	private final String icon = "icon";
+	private final String user = "user";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://www.codacy.com/app/joshiste/spring-boot-admin/file/2396352067/issues/source?bid=3175464&fileBranchId=3231081#l27)